api/logs/views.py

```diff
@@ -220,7 +221,7 @@ class NodeLogContributors(JSONAPIBaseView, generics.ListAPIView, ODMFilterMixin,
     required_read_scopes = [CoreScopes.USERS_READ]
     required_write_scopes = [CoreScopes.NULL]
 
-    serializer_class = UserSerializer
+    serializer_class = UnregisteredContributorSerializer
```

Comment:
Added test. 

I am checking is_anonymized in the contributors field within the params, so even if someone changes this later and params are shown in anonymous links, the contributors are still hidden.

So here, I've temporarily added 'params' to non_anonymized_fields and the contributors return an empty list.

![screen shot 2016-04-22 at 1 05 31 pm](https://cloud.githubusercontent.com/assets/9755598/14748981/e9e560e6-088a-11e6-8cbd-e3f5fbea24fe.png)